projects/mercury/src/metadata/views/MetadataViewTableContainer.js

```diff
@@ -353,28 +359,35 @@ export const MetadataViewTableContainer = (props: MetadataViewTableContainerProp
                     {renderMetadataViewTable()}
                 </TableContainer>
                 <div className={classes.footerButtonDiv}>
-                    <Button
-                        color="primary"
-                        className={classes.exportButton}
-                        onClick={exportTable}
-                        variant="contained"
-                        endIcon={<GetAppIcon fontSize="small" />}
-                        disabled={checkedCount === 0}
-                    >
-                        Download selection ({checkedCount})
-                    </Button>
-                    {exportToAnalysisEnabled && (
-                        <ProgressButton active={exportToAnalysisLoading}>
-                            <Button
+                    <div className={classes.footerCountDiv}>
+                        export items: {checkedCount}
```

Comment:
Or "Selected rows:"?
Could you use Typography component here? I thing it would look better if this text and "Rows per page:" text that is next to it have the same styling (color, size, font):
![Screenshot from 2023-03-09 15-43-43](https://user-images.githubusercontent.com/20908736/224060047-7ccb1ad1-3145-4e1e-8650-e4037938c919.png)

Reply:
and when the left margin is the same as for the checkboxes above